pkg/podexecutor/staticpod.go

```diff
@@ -127,14 +128,18 @@ func (s *StaticPodConfig) APIServer(ctx context.Context, etcdReady <-chan struct
 			args = append(args[:i], args[i+1:]...)
 		}
 	}
+	files := []string{etcdNameFile(s.DataDir)}
```

Comment:
I think we want to reverse this so we check to see if disabled and if it's not, we assign, rather than assigning and then emptying. Here and below.

Reply:
fixed